apps/nextjs/src/components/AppComponents/Chat/chat-panel.tsx

```diff
@@ -35,35 +35,48 @@ export function ChatPanel({ isDemoLocked }: Readonly<ChatPanelProps>) {
 
   const { trackEvent } = useAnalytics();
 
+  const sidebar = useSidebar();
+  const lessonPlanTracking = useLessonPlanTracking();
+
   const handleSubmit = useCallback(
```

Comment:
Half of this logic was in PromptForm, so I listed it up to here to be in one place